server/src/test/java/io/druid/server/initialization/JettyTest.java

```diff
@@ -360,4 +423,33 @@ public Response exception(
       throw new IOException();
     }
   }
+
+  public static class DummyAuthFilter implements Filter {
```

Comment:
would it make sense to make this a fully supported filter, so we can have a reference implementation for request filters?